modules/clusterSetup.js

```diff
@@ -249,3 +249,36 @@ export function checkForMultipleNodePools(clusterDetails) {
     details: details
   }
 }
+
+//
+// Checks if Azure Policy is enabled on Cluster
+//
+export function checkForAzurePolicy(clusterDetails) {
+
+  console.log(chalk.white("Checking for Azure Policy on cluster..."));
+
+  let details = [];
+
+  // Check if Azure policy is enabled
+  var azurepolicyConfigured = clusterDetails.addonProfiles.azurepolicy.enabled;
```

Comment:
I tested with a new AKS cluster and it was there for me. I have never node node is before so would not be too sure how to add a null-check.

Reply:
I'd probably do something like this:

```
var azurePolicy = clusterDetails.addonProfiles.azurepolicy;
var azurePolicyConfigured = azurePolicy && azurePolicy.enabled;
```

That way if azurepolicy is undefined on addonProfiles it will short circuit and not check for the enabled property on it. Otherwise if it's undefined (which it may be on older clusters) then it would error out trying to reference 'enabled' on a null object.

Reply:
Thanks for that. I will update pull request after doing some testing.

Reply:
Thats added in and tested ok with my clusters.